EventsExpress/DTO/EventDto.cs

```diff
@@ -37,5 +37,7 @@ public class EventDto
         public IEnumerable<CategoryDto> Categories { get; set; }
 
         public IEnumerable<UserPreviewDto> Visitors { get; set; }
+
+        public IEnumerable<InventoryDto> Inventories { get; set; }
```

Comment:
If this model is used in a preview only - I am not sure that we need inventories here. Please double check and remove it if not required.

Reply:
EventDto used in edit method only